src/playlist.js

```diff
@@ -321,19 +333,19 @@ export const playlistEnd = function(playlist, expired, useSafeLiveEnd, liveEdgeP
 
   expired = expired || 0;
 
-  let lastSegmentTime = intervalDuration(
+  let lastSegmentEndTime = intervalDuration(
```

Comment:
A more descriptive name as it isn't the start time, but the end time we are getting here.